drake/systems/Simulation.h

```diff
@@ -20,8 +20,15 @@ struct SimulationOptions {
   double initial_step_size;
   double timeout_seconds;
 
+  /**
+   * Whether to issue a warning if the simulation's real-time semantics are
+   * violated.
+   */
```

Comment:
A naive reading of this comment could suggest that, when set to false, the simulation will silently keep chugging along when realtime is violated (whereas really, it raises an exception).